.github/workflows/rustfmt.yml

```diff
@@ -1,5 +1,9 @@
-on: [push]
+# When pushed to `master`, run `cargo +nightly fmt --all` and commit any changes.
 name: rustfmt
+on:
+  push:
+    branches:
+      - master
```

Comment:
nit: we can probably lift this down into the commit action, i believe.
may at least try to run rustfmt everywhere possible?

Reply:
Changed to run this when pushed to any branch in your repo.